kibana-reports/.cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
where is this used?

Reply:
Not used, this was added by accident when adding the `.cypress` directory, will remove